library/Falcon/Handler/WPMail.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * WP Mail Handler
+ *
+ * @author Joe Hoyle <joe@huamanmade.co.uk>
+ * @package Falcon
+ * @subpackage Handlers
+ */
+class Falcon_Handler_WPMail implements Falcon_Handler {
+
+	public function __construct( $options ) {
+
+	}
+
+	public function check_inbox() {}
+
+	public function send_mail( $users, Falcon_Message $message ) {
+		$options = $message->get_options();
+
+		$from = Falcon::get_from_address();
+		$author = $message->get_author();
+
+		$messages = array();
+		foreach ( $users as $user ) {
+
+			$headers = array(
+				'From'         => sprintf( '%s <%s>', $author, $from ),
+				'Reply-To'     => sprintf( '%s', $message->get_reply_address( $user ) ),
+				'Content-Type' => 'text/html; charset=UTF-8',
```

Comment:
This should probably use the charset option instead.

Reply:
As it happens, wp_mail will set the chartset, so I just removed the directive